app/components/subcase/subcase-titles-panel/subcase-titles-panel-edit.js

```diff
@@ -49,6 +70,7 @@ export default class SubcaseTitlesPanelEdit extends Component {
     if (this.confidentialChanged && this.args.subcase.confidential) {
       yield this.pieceAccessLevelService.updateDecisionsAccessLevelOfSubcase(this.args.subcase);
       yield this.pieceAccessLevelService.updateSubmissionAccessLevelOfSubcase(this.args.subcase);
+      yield this.removeFromNewsletter.perform();
```

Comment:
bit of insider knowledge. 
The term "newsletter" in terms of the actual model was replaced to "news-Item / newsItem" a while ago.
some routes and services still have the name but used mainly in reference to the entire collection of newsItems for a meeting.
so a "newsletter" has a bunch of "newsItems"
There are still some "incorrect" names floating around, but we should avoid reintroducing more.